#Define The goal
uint32 dishwasher_id
---
#Define the result
uint32 total_dishes_cleaned
---
float32 percent_complete